pom.xml

```diff
@@ -876,6 +876,52 @@
             </build>
         </profile>
 
+        <profile>
```

Comment:
Can we add a README? Something like what we have for stability tests will do https://github.com/aws/aws-sdk-java-v2/tree/master/test/stability-tests